.drone.yml

```diff
@@ -5,7 +5,7 @@ type: kubernetes
 
 services:
   - name: docker
-    image: docker:20.10-dind
+    image: ghcr.io/smira/docker:20.10-dind-hacked
```

Comment:
do we do privileged somewhere? shouldn't be needed unless we do that

Reply:
I think it wasn't working without privileged.

Reply:
yep, I remember correctly:

```
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--
127 | destroying deployed cluster
128 | Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
129 | make: *** [Makefile:120: integration-test] Error 1
```